catalog-info.yaml

```diff
@@ -1115,6 +1115,41 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: beats-aws-tests-daily
+  description: 'Scheduled runs of Beats AWS tests'
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-aws-tests-daily
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: beats-aws-tests-daily
+      description: ':alarm_clock: Scheduled runs of Beats AWS tests'
+    spec:
+      repository: elastic/beats
+      pipeline_file: ".buildkite/daily-aws-pipeline.yml"
+      ranch_configuration: "main 8.* 7.17"
```

Comment:
There is a typo here, should `branch` instead of `ranch` but actually we don't need this line at all: the declaration of which branch it will be executed against are declared in the parent scheduler pipeline (and can be influenced with environment variables (e.g. https://github.com/elastic/logstash/blob/758098cdcd898bf6a2f1d132cb30aa6b519f55e7/catalog-info.yaml#L572)